llvm/test/CodeGen/AArch64/call-site-info-typeid.ll

```diff
@@ -0,0 +1,29 @@
+;; Tests that call site type ids can be extracted and set from type operand
+;; bundles.
+
+;; Verify the exact typeId value to ensure it is not garbage but the value
+;; computed as the type id from the type operand bundle.
+; RUN: llc --call-graph-section -mtriple aarch64-linux-gnu < %s -stop-before=finalize-isel -o - | FileCheck %s
```

Comment:
Why before finalize-isel and not after? Why does this need to check intermediate mir at all?